scripts/Inelastic/Direct/RunDescriptor.py

```diff
@@ -1339,7 +1340,7 @@ def _split_ws_name(self,ws_name):
 # Hell knows how to redefine these warnings or if they are valid or not
 #pylint: disable=W0141
 #pylint: disable=W0110
-            self._ws_cname = part_ind + filter(lambda c: not c.isdigit(), name)
+            self._ws_cname = part_ind + ''.join(re.findall('\d+', name))
```

Comment:
Whoops I missed the `not` and assumed it was the same the other `filter` cases I changed.